socorro/signature/siglists/irrelevant_signature_re.txt

```diff
@@ -207,6 +209,7 @@ RustMozCrash
 _sigtramp
 _security_check_cookie
 __security_check_cookie
+SegmentImpl
```

Comment:
This shows up in the signature like `mozilla::SegmentedVector<T>::SegmentImpl<T>::~SegmentImpl`, so this line has no effect on signature generation.

However, you added `mozilla::SegmentedVector` to the prefix list in this PR. That does cause the signature to change. For example:

```
app@socorro:/app$ socorro-cmd signature 46954e0a-21c6-4e2b-b39b-2f5350230717
Crash id: 46954e0a-21c6-4e2b-b39b-2f5350230717
Original: nsCOMPtr<T>::~nsCOMPtr | mozilla::SegmentedVector<T>::SegmentImpl<T>::PopLast
New:      nsCOMPtr<T>::~nsCOMPtr | mozilla::SegmentedVector<T>::SegmentImpl<T>::PopLast | mozilla::SegmentedVector<T>::PopLastN | mozilla::dom::DeferredFinalizerImpl<T>::DeferredFinalize
Same?:    False
```
```

Reply:
I've update this in a new patch.